packages/sit-onyx/src/components/OnyxMoreList/TestWrapper.ct.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts" setup>
+import OnyxNavButton from "../OnyxNavBar/modules/OnyxNavButton/OnyxNavButton.vue";
+import { NAV_BAR_MORE_LIST_INJECTION_KEY } from "../OnyxNavBar/types";
+import OnyxMoreList from "./OnyxMoreList.vue";
+import type { MoreListSlotBindings } from "./types";
+
+const props = defineProps<{
+  /**
+   * Number of components to show. Can also be decimal.
+   */
+  count?: number;
```

Comment:
Why do we need decimal?

Reply:
This is just for the Playwright test so you can e.g. pass 2.5 to test that then only 2 are fully shown and the third is hidden :)